modules/installation-overview.adoc

```diff
@@ -77,15 +65,25 @@ of new certificates or keys, after installation.
 == Available platforms
 
 In {product-title} {product-version}, you can install a cluster that uses
-installer-provisioned infrastructure on only Amazon Web Services (AWS).
+installer-provisioned infrastructure on the following platforms:
+
+* Amazon Web Services (AWS)
+* Google Cloud Platform (GCP)
+* Microsoft Azure
+* OpenStack
```

Comment:
Shouldn't this be "Red Hat OpenStack Platform" since you point out Google/Microsoft, etc. for the others?

Reply:
@bergerhoffer, I am not sure. The OSP docs themselves never use the full product name, and I am not sure if it's restricted to just the Red Hat version. @maxwelldb, thoughts?

Reply:
AFAIK, (virtually?) none of it is limited to the Red Hat version.

Reply:
Ack. However, the release notes specify Red Hat's OpenStack, so we should probably get confirmation and sync these up.

[1] https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html

Reply:
Engineering believes that our content applies to generic OpenStack as well as RHOSP, though we only support RHOSP. Verifying what PM wants, though, given that this is in the context of OCP docs. Could take beyond today.

Reply:
PM wants to use Red Hat OpenStack Platform (RHOSP).

Reply:
Then this PR is good. Thank you for pointing this out @bergerhoffer!